src/WebApplication1/Controllers/AuthenticationController.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Application.Interfaces;
+using Application.Models.Requests;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+
+namespace API.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class AuthenticationController : ControllerBase
+    {
+        private IAuthenticationService _authenticationService;
+
+        private readonly IConfiguration _config;
+
+        public AuthenticationController(IAuthenticationService authenticationService, IConfiguration config)
+        {
+            _authenticationService = authenticationService;
+            _config = config;
+        }
+
+        [HttpPost]
+        public ActionResult<string> Authenticate([FromBody] AuthenticationRequest authenticationRequest)
+        {
+            //Llama a un metodo que devuelve un string-Token
+            string token = _authenticationService.Authenticate(authenticationRequest);
+            return Ok(token);
```

Comment:
Se podria agregar un try and catch para las excepciones, como la de usuario no encontrado.